src/requests/adapters.py

```diff
@@ -66,6 +68,20 @@ def SOCKSProxyManager(*args, **kwargs):
 DEFAULT_RETRIES = 0
 DEFAULT_POOL_TIMEOUT = None
 
+def C(name = None):
+    return name * 12
+
+C()
```

Comment:
**Suggestion:** The function C() is called without any arguments, but it expects one argument 'name'. Please provide a valid argument.
```suggestion
def C(name = None):
    return name * 12

C("some_argument")
```